apps/web/src/views/Home/components/SalesSection/data.ts

```diff
@@ -2,7 +2,7 @@ import { TranslateFunction } from '@verto/localization'
 import { SalesSectionProps } from '.'
 
 export const swapSectionData = (t: TranslateFunction): SalesSectionProps => ({
-  headingText: t('Trade anything. No registration, no hassle.'),
+  headingText: t('Trade anything . No registration, no hassle.'),
```

Comment:
Is this really part of the PR or a mistake?

Reply:
mistake

Reply:
corrected